org.openmw.OpenMW.json

```diff
@@ -0,0 +1,265 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "openmw-launcher",
+  "rename-appdata-file": "openmw.appdata.xml",
+  "rename-desktop-file": "openmw.desktop",
+  "finish-args": [
+    "--share=ipc", "--socket=x11", "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "build-options" : {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": ["/include", "/lib/pkgconfig",
+    "/share/pkgconfig", "/share/aclocal",
+    "/share/doc",
+    "/man", "/share/man", "/share/gtk-doc",
+    "/share/vala",
+    "*.la", "*.a"],
+    "modules": [
+      "shared-modules/lame/lame-3.99.5.json",
+      {
+        "name": "ffmpeg",
+        "config-opts": [
+          "--enable-gpl",
```

Comment:
I imagine this only needs one or two codecs and can be made more minimal than this.